### PR TITLE
Added my version of Hello World C++ program.

### PR DESCRIPTION
**Pull request by andrewchanintuit**
This is the first version of Hello World in C++.

### CODE REVIEW

File: achan_hello_world.cc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Header comment
```
Comment (andrewchanintuit) [inline achan_hello_world.cc:1]: Nice comment.

**Comment by andrewchanintuit** [comment]
Ready for merge pull request!

File: achan_hello_world.cc [hunk 2]
```diff
@@ -1,4 +1,5 @@
 // Header comment
+// Even nicer header comment
```
Comment (andrewchanintuit) [inline achan_hello_world.cc]: Even nicer!

Comment (andrewchanintuit) [inline achan_hello_world.cc:2]: Even nicer comment!

**Comment by thomp812** [comment]
you're having way too much fun with this...

**Comment by andrewchanintuit** [comment]
Ha.  It's because I needed to practice it again since @garyl553 screwed me up during class.  He updated my empty file check-in, then when I tried to commit the file with real C++ code, git complained that I needed to merge and I got all screwed up! :)

**Pull request merged**